### PR TITLE
FeedOn - Front end

### PR DESCRIPTION
**Pull request by anushadatta**
Coded the front end for landing page, donation form and template for donations inbox.  Also re-organized the code file structure.

Screen Shot 2020-07-14 at 12 08 19 AM

Screen Shot 2020-07-14 at 12 05 55 AM

Screen Shot 2020-07-14 at 12 06 19 AM

### CODE REVIEW

File: src/main/webapp/donation.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <option value="1">Vegetarian</option>
+            <option value="2">Non-Vegetarian</option>
+            <option value="3">Halal</option>
+        </select>
+        <br>
+
+        <!-- Date & Time -->
+        <br>
+        <label><b>Specify a date and time for donation:</b></label>
+        <input type="datetime-local" id="birthdaytime" name="birthdaytime" required>
```
Comment (rashikaranpuria) [inline src/main/webapp/donation.html:50]: Why is the id birthdaytime?

File: src/main/webapp/js/client.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+function listAllFeeds() {
+    getFeeds().then((feeds) => {
+        dataElement.innerHTML = "";
+        feeds.forEach((feed) => {
+            const div = document.createElement("div");
+
+            const header = document.createElement("h3");
+            header.textContent = `${feed.restaurant.name} & ${feed.charity.name}`;
+
+            const contents = document.createElement("p");
```
Comment (rashikaranpuria) [inline src/main/webapp/js/client.js:15]: SHould we call it description

File: src/main/webapp/donation.html (hunk 1 above)
Comment (anushadatta) [reply src/main/webapp/donation.html:50]: Big oops, I fixed this in my consequent PR.

File: src/main/webapp/js/client.js (hunk 2 above)
Comment (anushadatta) [reply src/main/webapp/js/client.js:15]: Yes, this JS code segment was written by @anh2111htd. @KhawYewOnn and I already suggested this change, which should now be reflected in consequent PRs.

**Pull request merged**